### PR TITLE
Expanded test scripts for GetVD(FuelRange)

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
- boundary values
 - out of boundary values
 - missed params
 - invalid types
 - missed fuelRange parameter in HMI response

"GitHub issue link" ~ "https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0072-New-vehicle-data-FuelRange.md"

### CODE REVIEW

File: test_scripts/API/VehicleData/GetVehicleData/003_GetVD_fuelRange_boundary_values.lua [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  local mobileSession = common.getMobileSession(self, 1)
+  local cid = mobileSession:SendRPC(rpc.name, rpc.params)
+  EXPECT_HMICALL("VehicleInfo." .. rpc.name, rpc.params)
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS",
+       { fuelRange = fuelRangeValue })
+    end)
+  mobileSession:ExpectResponse(cid,
+    { success = true, resultCode = "SUCCESS", fuelRange = fuelRangeValue })
+  commonTestCases:DelayedExp(500)
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/GetVehicleData/003_GetVD_fuelRange_boundary_values.lua]: @GetmanetsIrina Please clarify the reason of usage DelayedExp() function here?

File: test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+-- User story: TO ADD !!!
+-- Use case: TO ADD !!!
+--
+-- Requirement summary:
+-- [GetVehicleData] As a mobile app wants to send a request to get the details of the vehicle data
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid GetVehicleData to SDL and this request is allowed by Policies
+-- 2) HMI sends boundary values of parameters in response
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua]: @GetmanetsIrina Please correct description according to script

File: test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+local function processRPCSuccess(fuelRangeValue, self)
+  local mobileSession = common.getMobileSession(self, 1)
+  local cid = mobileSession:SendRPC(rpc.name, rpc.params)
+  EXPECT_HMICALL("VehicleInfo." .. rpc.name, rpc.params)
+  :Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS",
+     { fuelRange = fuelRangeValue })
+  end)
+  mobileSession:ExpectResponse(cid,
+    { success = true, resultCode = "SUCCESS", fuelRange = fuelRangeValue })
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua]: @GetmanetsIrina Is this correct response to App in case of missed mandatory parameters in HMI response?

Comment (GetmanetsIrina) [reply test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua]: @dboltovskyi  yes, all parameters of fuelRange in response are not mandatory, so expected success result.

File: test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua]: Done

File: test_scripts/API/VehicleData/GetVehicleData/003_GetVD_fuelRange_boundary_values.lua (hunk 1 above)
Comment (dboltovskyi) [reply test_scripts/API/VehicleData/GetVehicleData/003_GetVD_fuelRange_boundary_values.lua]: @GetmanetsIrina Accepted

File: test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua (hunk 2 above)
Comment (dboltovskyi) [reply test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua]: @GetmanetsIrina There is no new commits in this pull request

File: test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua (hunk 3 above)
Comment (dboltovskyi) [reply test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua]: @GetmanetsIrina Accepted

File: test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua]: @dboltovskyi, please check 0d91732b742b57867fa93347d9a12b3d67ccfc03 .
